### PR TITLE
Remove conditional-compile lines for file-sharing-teams-interop

### PR DESCRIPTION
**Pull request by JamesBurnside**
# What

Remove conditional-compile lines for file-sharing-teams-interop

# Why

Housekeeping. Cleaning up lines now that feature is in stable release

# How Tested

CI only

### CODE REVIEW

**Comment by Leah-Xia-Microsoft** [comment]
@palatter could you help reviewing this PR to make sure CCs are removed as expected? Thanks

File: packages/chat-component-bindings/src/messageThreadSelector.ts [hunk 1]
```diff
@@ -73,7 +70,7 @@ const extractAttachmentMetadata = (metadata: Record<string, string>): Attachment
     return [];
   }
 };
-/* @conditional-compile-remove(file-sharing-teams-interop) @conditional-compile-remove(file-sharing-acs) */
+
```
Comment (alkwa-msft) [inline packages/chat-component-bindings/src/messageThreadSelector.ts:73]: do we want to be removing the empty lines as well?

Comment (JamesBurnside) [reply packages/chat-component-bindings/src/messageThreadSelector.ts:73]: I've tried to be consistent with what is already in each respective file throughout

**Comment by palatter** [comment]
File-sharing-acs is still in beta, so we still need those CC tags

**Review by palatter — APPROVED** [review]
Thanks for cleaning up these CCs

**Review by jpeng-ms — APPROVED** [review]

**Pull request merged**